kubernetes/zenko/charts/zenko-nfs/values.yaml

```diff
@@ -1,6 +1,9 @@
 # Default values for zenko-nfs.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
+
+enabled: false
```

Comment:
Don't need at this level

Reply:
It does in order to pass `helm lint --strict`. Changed default value to `true`.